pkg/controllers/dynakube/logmonitoring/daemonset/volumes.go

```diff
@@ -22,14 +22,10 @@ const (
 	dtLogVolumeMountPath = "/var/log/dynatrace"
```

Comment:
Can confirm this way it does not work. Anyone has an idea how we should proceed from now on? Because I think this mount is a requirement that was given to usw